WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/list/OrderListFragment.kt

```diff
@@ -774,4 +787,58 @@ class OrderListFragment :
             }
         }
     }
+
+    private fun displayQuickOrderWIPCard(show: Boolean) {
+        if (!show || feedbackState == FeatureFeedbackSettings.FeedbackState.DISMISSED) {
+            binding.quickOrderWIPcard.isVisible = false
+            return
+        }
+
+        val isEnabled = AppPrefs.isQuickOrderEnabled
+        @StringRes val messageId = if (isEnabled) {
+            R.string.orderlist_quickorder_wip_message_enabled
+        } else {
+            R.string.orderlist_quickorder_wip_message_disabled
+        }
+
+        binding.quickOrderWIPcard.isVisible = true
+        binding.quickOrderWIPcard.initView(
+            getString(R.string.orderlist_quickorder_wip_title),
+            getString(messageId),
+            onGiveFeedbackClick = { onGiveFeedbackClicked() },
+            onDismissClick = { onDismissWIPCardClicked() },
+            showFeedbackButton = isEnabled
+        )
+    }
+
+    private fun onGiveFeedbackClicked() {
+        AnalyticsTracker.track(
+            Stat.FEATURE_FEEDBACK_BANNER,
+            mapOf(
+                AnalyticsTracker.KEY_FEEDBACK_CONTEXT to AnalyticsTracker.VALUE_QUICK_ORDER_FEEDBACK,
+                AnalyticsTracker.KEY_FEEDBACK_ACTION to AnalyticsTracker.VALUE_FEEDBACK_GIVEN
+            )
+        )
+        registerFeedbackSetting(FeatureFeedbackSettings.FeedbackState.GIVEN)
+        NavGraphMainDirections
+            .actionGlobalFeedbackSurveyFragment(SurveyType.QUICK_ORDER)
+            .apply { findNavController().navigateSafely(this) }
+    }
+
+    private fun onDismissWIPCardClicked() {
+        AnalyticsTracker.track(
+            Stat.FEATURE_FEEDBACK_BANNER,
+            mapOf(
+                AnalyticsTracker.KEY_FEEDBACK_CONTEXT to AnalyticsTracker.VALUE_QUICK_ORDER_FEEDBACK,
+                AnalyticsTracker.KEY_FEEDBACK_ACTION to AnalyticsTracker.VALUE_FEEDBACK_DISMISSED
+            )
+        )
+        registerFeedbackSetting(FeatureFeedbackSettings.FeedbackState.DISMISSED)
+        displayQuickOrderWIPCard(false)
+    }
+
+    private fun registerFeedbackSetting(state: FeatureFeedbackSettings.FeedbackState) {
+        FeatureFeedbackSettings(FeatureFeedbackSettings.Feature.QUICK_ORDER.name, state)
+            .run { FeedbackPrefs.setFeatureFeedbackSettings(TAG, this) }
```

Comment:
Sorry to jump in a PR I'm not reviewing, but just for awareness, nothing really important, you can use the `registerItselfWith` in case you want to make this operation shorter, or control/reference only the `FeatureFeedbackSettings` class:

```suggestion
        FeatureFeedbackSettings(
            FeatureFeedbackSettings.Feature.QUICK_ORDER.name, 
            state
        ).registerItselfWith(TAG)
```